package.json

```diff
@@ -22,9 +22,12 @@
   "scripts": {
     "build": "tsc --build tsconfig.json",
     "lint": "polkadot-dev-run-lint",
-    "start": "ts-node src/index.ts"
+    "start": "ts-node src/index.ts",
+    "generate:acala": "npx parachain-launch generate --config=configs/acala.yml --y",
```

Comment:
let’s move those to here https://github.com/AcalaNetwork/Acala/blob/master/launch/package.json

Reply:
Done